travis_osx_steps.sh

```diff
@@ -2,6 +2,18 @@
 # Wheel build, install, run test steps on OSX
 set -e
 
+if [ "$PLAT" == "arm64" ] || [ "$PLAT" == "universal2" ]; then
+  if [[ "$(xcrun -show-sdk-version)" == 10.*  ]]; then
+    if ([ "$GITHUB_WORKFLOW" != "" ] || [ "$PIPELINE_WORKSPACE" != "" ]) && [ -d /Applications/Xcode_12.2.app ]; then
+      sudo xcode-select -switch /Applications/Xcode_12.2.app
```

Comment:
```suggestion
    if ([ "$GITHUB_WORKFLOW" != "" ] || [ "$PIPELINE_WORKSPACE" != "" ]) && [ -d /Applications/Xcode_12.*.app ]; then
      sudo xcode-select -switch /Applications/Xcode_12.*.app
```
Testing, I see no reason why this shouldn't work.